### ISSUE TITLE
Option for different VPN provider

### ISSUE THREAD

**Issue by MHP47** [tom472/mediabox#92]
PIA is going through some changes, being acquired by a company with a shoddy past.

See [here](https://old.reddit.com/r/PrivateInternetAccess/comments/dz2w53/our_merger_with_kape_technologies_addressing_your/)

Is there any plans/roadmap for offering alternative VPN providers during install?

P.S. Awesome repo, love my setup, been using for nearly a year now

**Comment by tom472** [comment]
Yes - I have seen the info of the purchase and that the new company is a little less than stellar. I am taking a little bit of to to see where all the chips fall and weighing options.

Yes there will likely be an new/alternate VPN provider option coming very soon, please stay tuned.

And @MHP47 Thank you very much for the nice words about the project.

**Comment by Faddermakker** [comment]
Did you make any progress since november?

**Comment by tom472** [comment]
Currently I haven't switched away from PIA.
However even if I don't, I do want to try and add options.
Unfortunately for a while I have been very tired up from work and have not had much extra time to put into Mediabox recently.
Hoping I will get some time available soon.
And thank you very much for the nice comment on the project!

**Comment by MHP47** [comment]
@tom472 I am planning to make the move across to [Mullvad](https://mullvad.net/en/)
I will look to create a change, and submit a PR for it, but no idea on timeline :) But I'll try
